lib/taste_tester/config.rb

```diff
@@ -56,6 +56,13 @@ module Config
     chef_config 'client.rb'
     my_hostname nil
 
+    if Gem.win_platform?
```

Comment:
rather than overwrite these, I'd do:

```
# platform agnostic settings here
if Gem.win_platform?
  # windows-specific settings here
else
  # non-windows version of above here
end
```